daemon/src/db.rs

```diff
@@ -323,22 +340,204 @@ impl Connection {
             from
                 cfds
             where not exists (
+                select id from EVENTS as events
+                where cfd_id = cfds.id and
```

Comment:
Should this be? 

```suggestion
                where events.cfd_id = cfds.id and
```

I'm not sure how smart sql is but above you said `id as cfd_id` and then access it here again with `cfds.id` which means this is always true? :D

Reply:
I will work without qualifying the table name here, because `cfd_id` only exists in `events`, but we should probably generally qualify the tables by name to make this cleaner.